### DISCUSSION TITLE
Lock mouse to middle of screen

### DISCUSSION THREAD

**Discussion by discoveryX64** [rustdesk/rustdesk#5922] [Ideas]
Title. This is meant for games and stuff that usually has your mouse locked to the middle, but doesn't when on rustdesk. I'm proposing a feature that would lock it to the middle. (Like a hotkey)

**Comment by discoveryX64** [comment]
Whoops, already did this 🤣

**Reply by jacob-squires** [reply]
I'm still trying to find a way to lock the mouse to the RustDesk window. When playing games when I move the mouse travels past the RustDesk window and loses focus. Can't really play games with RustDesk atm with this. Did you happen to fix this somehow? I haven't see any options to enable a Lock Mouse.

**Reply by discoveryX64** [reply]
No sadly
